Acsone addons asynchronous batch mailings

JonathanNEMRY opened the pull request:
This Module allows to send email by an asynchronous way.
It also provides a way to split huge mailing.
Those features are configurable into ir.parameter

Commit:
[acsone-addons][asynchronous_batch_mailings][NEW]
This Module allows to send email by an asynchronous way.
It also provides a way to split huge mailing.
[acsone-addons][CHG] add dependency on connector in .travis.yml

Commit:
[distribution_list][FIX] add a context with no tracking to avoid mail.thread problems
[asynchronous_batch_mailings][FIX] test context before processing. Check len() of active_ids

JonathanNEMRY commented:
Any news about this PR?

lmignon commented on asynchronous_batch_mailings/__init__.py:
```diff
@@ -0,0 +1,29 @@
+#    Authors: Nemry Jonathan
+#    Copyright (c) 2014 Acsone SA/NV (http://www.acsone.eu)
+#    All Rights Reserved
+#
```
all rights reserved isn't relevant. The licence can be shortened..

``` python
##############################################################################
#
#    Authors: Nemry Jonathan
#    Copyright (c) 2014 Acsone SA/NV (http://www.acsone.eu)
#
#    This program is free software: you can redistribute it and/or modify
#    it under the terms of the GNU Affero General Public License as
#    published by the Free Software Foundation, either version 3 of the
#    License, or (at your option) any later version.
#
#    This program is distributed in the hope that it will be useful,
#    but WITHOUT ANY WARRANTY; without even the implied warranty of
#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
#    GNU Affero General Public License for more details.
#
#    You should have received a copy of the GNU Affero General Public License
#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
#
##############################################################################
```

lmignon commented:
:+1: (Code review)

anthony-muschang commented:
:+1:

Olivier-LAURENT committed:
[CHG] travis script to use $VERSION
[FIX] remove duplicate key in dictionary
[FIX] specify a relative path to import tests

Olivier-LAURENT commented:
:+1:

Olivier-LAURENT committed:
[CHG] code review

Olivier-LAURENT committed:
[FLK8]

lmignon commented:
Everything looks fine now :+1:
